CHANGES.rst

```diff
@@ -306,6 +306,9 @@ skymatch
 - Updated code in ``skymatch.region.py`` with latest improvements and bug fixes
   from ``stsci.skypac.regions.py``. [#6451]
 
+- Updated documentation to clarify details of flat-fielding versus distortion
+  corrections [#6546]
```

Comment:
I do not believe documentation updates need a changelog entry. What is the policy with regard to this in this repository, @hbushouse

Reply:
Yes, it's fine (and encouraged) to have change log entries for doc updates.